app/src/main/java/com/github/multimatum_team/multimatum/activity/GroupsActivity.kt

```diff
@@ -41,4 +47,39 @@ class GroupsActivity : AppCompatActivity() {
             adapter.setGroups(groups)
         }
     }
+
+    fun addGroup(view: View){
+        val alertDialogBuilder = AlertDialog.Builder(this)
+        alertDialogBuilder.setTitle("Create group")
+        alertDialogBuilder.setMessage("Group name :")
+
+        //add editable text box for group name
+        val input = EditText(this@GroupsActivity)
+        val lp = LinearLayout.LayoutParams(
+            LinearLayout.LayoutParams.MATCH_PARENT,
+            LinearLayout.LayoutParams.MATCH_PARENT
+        )
+        input.layoutParams = lp
+        alertDialogBuilder.setView(input)
+
+        //add cancel and create button
+        alertDialogBuilder.setPositiveButton("Create") { dialog, _ ->
+            createGroup(dialog, input)
+        }
+        alertDialogBuilder.setNegativeButton("Cancel") { dialog, _ ->
+            dialog.dismiss()
+        }
+        alertDialogBuilder.create()
```

Comment:
This function is not necessary as the show() call this function too but this a nitpick as it doesn't change anything